core/src/main/java/io/questdb/cairo/wal/seq/TableTransactionLog.java

```diff
@@ -206,7 +211,7 @@ void open(Path path) {
             txnMem.jumpTo(0L);
             txnMem.putInt(WAL_FORMAT_VERSION);
             txnMem.putLong(0L);
-            txnMem.putLong(0L);
+            txnMem.putLong(tableCreateTimestamp);
```

Comment:
just in case
```suggestion
            assert tableCreateTimestamp != Long.MIN_VALUE;
            txnMem.putLong(tableCreateTimestamp);
```

Reply:
fixed in: https://github.com/questdb/questdb/pull/3637/commits/c70d5416fb61308635c748fb6ce8ef554098bc95